Common++/header/SystemUtils.h

```diff
@@ -1,12 +1,15 @@
 #pragma once
 
-#include <stdint.h>
+#include <cstdint>
 #include <string>
 #include <vector>
 
 /// @file
 
-#define MAX_NUM_OF_CORES 32
+enum
+{
+	MAX_NUM_OF_CORES = 32
+};
```

Comment:
Why replace a macro with enum? 🤔

Reply:
It is an auto-fix from https://clang.llvm.org/extra/clang-tidy/checks/modernize/macro-to-enum.html